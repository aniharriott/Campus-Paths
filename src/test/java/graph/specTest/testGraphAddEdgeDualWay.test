# Test Graph addEdge dual way file.
# Tests if given two nodes, two edges
# can point opposite directions when
# associated with those same two nodes.

# Create graph
CreateGraph graph1

# Add nodes
AddNode graph1 n1
AddNode graph1 n2

# Add edges
AddEdge graph1 n1 n2 e1
AddEdge graph1 n2 n1 e2