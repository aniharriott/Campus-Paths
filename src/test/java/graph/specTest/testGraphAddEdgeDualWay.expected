# Test Graph addEdge dual way file.
# Tests if given two nodes, two edges
# can point opposite directions when
# associated with those same two nodes.

# Create graph
created graph graph1

# Add nodes
added node n1 to graph1
added node n2 to graph1

# Add edges
added edge e1 from n1 to n2 in graph1
added edge e2 from n2 to n1 in graph1